spec/fixtures/legacy_app_no_template.js

```diff
@@ -0,0 +1,55 @@
+with( ZendeskApps.AppScope.create() ) {
```

Comment:
Yeah, all I changed is the manifest which has a noTemplate option for ticket_sidebar, because that's all I'm testing for.

I agree it's a big file for only that tiny change, to you suggest a different solution?